src/components/Button/Button.module.scss

```diff
@@ -1,11 +1,12 @@
 .button {
-  @apply rounded px-4 py-2 text-sm font-bold;
+  @apply w-[296px] h-[40px] rounded-lg px-4 py-2 text-sm font-bold;
 }
 
 .primary {
-  @apply bg-red-500 text-white hover:bg-blue-600;
+  @apply bg-blue-500 text-white hover:bg-blue-600; // No border for primary
```

Comment:
primary color should not be hard-coded in the common component CSS, it would be picked up from theme.